pkgs/development/python-modules/para/default.nix

```diff
@@ -0,0 +1,26 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+, nose
+}:
+
+buildPythonPackage rec {
+  pname = "para";
+  version = "0.0.8";
+
+  src = fetchPypi {
```

Comment:
why not `fetchFromGitHub`?

Reply:
Because unfortunately there are no releases/tags on the repo.